apps/web/pages/signup.tsx

```diff
@@ -366,7 +362,7 @@ export default function Signup({
                       if (username) {
                         // If username is present we save it in query params to check for premium
                         const searchQueryParams = new URLSearchParams();
-                        searchQueryParams.set("username", formMethods.getValues("username"));
+                        searchQueryParams.set("username", username);
```

Comment:
Because username can be nullish, use username variable which is checked for nullishness